src/System/ref/Compat/System.cs

```diff
@@ -4995,14 +4995,14 @@ public partial class CookieCollection : System.Collections.ICollection, System.C
         public CookieCollection() { }
         public int Count { get { return default(int); } }
         public bool IsReadOnly { get { return default(bool); } }
-        public bool IsSynchronized { get { return default(bool); } }
+        bool System.Collections.ICollection.IsSynchronized { get { return default(bool); } }
+        object System.Collections.ICollection.SyncRoot { get { return default(object); } }
```

Comment:
I did it like this just because this is how the actual implementation is today. Keep in mind this is only a reference assembly for a facade, the reference assembly where the implementation lives today is [this](https://github.com/dotnet/corefx/blob/master/src/System.Net.Primitives/ref/System.Net.Primitives.cs#L51-L52) and, as you can see, the ref is the same as mine.